### DISCUSSION TITLE
Advantage of BAUET Result System?

### DISCUSSION THREAD

**Discussion by ghost** [mdarikrayhan/BAUET_Result_System#6] [Q&A] [answered]
What are the Advantage of BAUET Result System?

**Comment by mdarikrayhan** [comment, marked as answer]
For Management:
1. With student result management system, teachers, professors, and staff can create result analysis reports.
2. Gradebook Management.
3.Exam Timetable Management.
4. Records marks and result on a single database.
5. Add & manage students and Declare Results.
6. Integrate-able with fee management.
7. Send/ publish results to parents and students online.
8. Record and search details of every exam/test, student wise/course wise.
9. Admit Card/ Hall Ticket Generation.
10. Calculate scores, percentages and grades.
